learning_service/notebooks/requirements.notebook.txt

```diff
@@ -64,8 +64,8 @@ httplib2==0.19.1
 idna==3.3
 importlib-metadata==4.11.4
 importlib-resources==5.7.1
-ipykernel==6.13.0
-ipython==8.3.0
+ipykernel
```

Comment:
Would be great to use requirements freeze to settle on the library versions for both learning and inference services.